mise.toml

```
@@ -2,7 +2,7 @@
 go = "1.25.4"
 "go:go.uber.org/mock/mockgen" = "v0.6.0"  # renovate: datasource=github-releases depName=uber-go/mock
 golangci-lint = "2.6.1"
-goreleaser = "v2.9.0"
+goreleaser = "v2.9.0"                     # renovate: datasource=github-releases depName=goreleaser/goreleaser
```

What about being explicit about the backend and specify the full path to the project?


```suggestion
ubi:goreleaser/goreleaser = "v2.9.0"                     # renovate: datasource=github-releases depName=goreleaser/goreleaser
```

I don't know if this is using ubi or go.

(I would even do the same for golangci-lint)